internal/cli/templates/service/cmd/grpc/main.go.tmpl

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	f "github.com/foundation-go/foundation"
+	ferr "github.com/foundation-go/foundation/errors"
+	pb "github.com/foundation-go/foundation/examples/clubchat/protos/chats"
```

Comment:
As I understand that's just an example and users will substitute their proto?

Reply:
correct